libkbfs/repeated_wait_group.go

```diff
@@ -20,6 +20,8 @@ type RepeatedWaitGroup struct {
 	lock     sync.Mutex
 	num      int
 	isIdleCh chan struct{} // leave as nil when initializing
+	paused   bool
```

Comment:
I think you can avoid the need for the `paused` boolean like so:

1) Have `ch` initialized to a channel
2) On Pause, close `ch` and set it to `nil`, or panic if it's already `nil`
3) On Resume, re-initialize `ch`, or panic if it's already re-initialized
4) In `WaitUnlessPaused`, if `ch` is `nil`, then we're paused so return immediately.
5) Otherwise, wait on `ch` as part of the select.

I guess `ch` should be named `onPauseCh` or something?